src/Fixer/Phpdoc/PhpdocTypesOrderFixer.php

```diff
@@ -169,7 +169,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
      */
     private function sortTypes(TypeExpression $typeExpression): array
     {
-        $normalizeType = static fn (string $type): string => Preg::replace('/^\\??\\\?/', '', $type);
+        $normalizeType = static fn (string $type): string => Preg::replace('/^\\(?\\??\\\\?/', '', $type);
```

Comment:
```suggestion
        $normalizeType = static fn (string $type): string => Preg::replace('/^\(*\??\\\?/', '', $type);
```

Reply:
I would leave variable name as is, because it's later used as invokable and it reads better. When it comes to regex, what's the difference between PR's one and yours, @mvorisek? If I see correctly you removed implicit escape backslashes, but also added support for multiple opening parentheses? If I'm right, it should be added to test cases too.

In general @smnandre it's a nice fix, totally should be supported 🙂.

Reply:
variable name - it took me a while to understand for what it is used - the result is only for compare, also `Fx` suffix is a good practice to name anonymous functions

regex - yes, it allows more parentheses and test should be added

Reply:
`feat` in my opinion, as sorting `(x` vs `x` was not implemented before and nothing was broken

Reply:
Thx for the feedback, i'll update and clean tomorrow !

Reply:
Regexp updated + tests to check double parenthesis added